lib/mcb.rb

```diff
@@ -234,6 +150,56 @@ def get_user_from_config
       user
     end
 
+    def apiv1_opts(**opts)
+      opts.merge! azure_env_settings_for_opts(**opts)
+
+      if requesting_remote_connection?(**opts)
+        opts[:url] = MCB::Azure.get_urls(**opts).first
+        opts[:token] = MCB::Azure.get_config(**opts)['AUTHENTICATION_TOKEN']
+      end
+
+      opts
+    end
+
+    def iterate_v1_endpoint(url:, endpoint:, **opts)
```

Comment:
just wondering can this be moved to it own, as this file is a bit big

this mcb file seems to forever dealing with doing prerequisites like, fetching azure/config/cli args then the actionable bits

Reply:
Sounds reasonable to me. Will to.

Reply:
I just had a look and I think that this refactoring should be a larger bit of work to move out all the API V1 related methods into a separate module. Good call, but I think it's worth putting into it's own PR.